recipes/onetbb/all/conandata.yml

```diff
@@ -15,6 +15,11 @@ sources:
     url: "https://github.com/oneapi-src/oneTBB/archive/v2021.3.0.tar.gz"
     sha256: "8f616561603695bbb83871875d2c6051ea28f8187dbe59299961369904d1d49e"
 patches:
+  "2021.9.0":
+    - patch_description: "fix building for emscripten"
+      patch_type: "portability"
+      patch_source: "https://github.com/oneapi-src/oneTBB/pull/1006.patch"
```

Comment:
I am asking in https://github.com/oneapi-src/oneTBB/pull/1006#issuecomment-1584894267 what is the status of this patch, it is not that trivial, so some insights from the original author could be useful.